lib/checkPullRequestReview.js

```diff
@@ -127,8 +127,9 @@ const handleApprovalByAllReviewers = async (context, pullRequest) => {
     const commentBody =
       'Hi @' +
       projectOwner +
-      ', this PR is ready to be merged. Before you merge it, please ' +
-      'make sure that there are no pending comments that require ' +
+      ', this PR is ready to be merged. Author of this PR ' +
+      'do not have permissions to merge this PR. Before you merge it, ' +
+      'please make sure that there are no pending comments that require ' +
       "action from the author's end. Thanks!";
```

Comment:
Could you change this to
```suggestion
      'action from the author's end. Thanks!';
```